Added description for ESXi Hypervisor (#701)
MichaIng/DietPi-Docs: docs/install.md

        ![DietPi VM creation screenshot](assets/images/VMwareESXi_04.jpg){: width="942" height="556" loading="lazy"}

    1. Tab **Select storage**: Choose the location where the VM shall reside. Then click **Next**.
    1. Tab **Deployment options**: Set options according to your preferences and click **Next**.
    1. Tab **Ready to complete**: Check dialog and click **Finish**.

    <font size="+2">3. First boot of the new VMware image</font>

    Depending on your settings the VM automatically starts up and is shown in the VM management dialog. Otherwise click on the VM name.

    ![vSphere VM management screenshot 2](assets/images/VMwareESXi_05.jpg){: width="1669" height="859" loading="lazy"}

    You can open a console of your VM just by clicking on the small VM screen output to open a console window. Klick into the new console window to redirect your keyboard to this console.

    !!! info "Install VM tools"

        To install VM tools in your ESXi machine, it is an option to do it with the `open-vm-tools` via:

        ```sh
        apt install open-vm-Tools
        ```

    !!! info "Boot times of a VM can be fast (less than 10 seconds)"

=== "Proxmox"

> 

Reviewer: You stole the `Introduction` header  header from the Proxmox tab 😄.

Author: Merge conflict...

    One big advantage of such a VM is that it needs only a couple of minutes coming to a running DietPi system.

    One of the options of a virtual machine appliance server is the freely available [__VMware vSphere Hypervisor__](https://www.vmware.com/products/vsphere-hypervisor.html/) which contains the [VMware ESXi Hypervisor](https://www.vmware.com/products/esxi-and-esx.html/).

    ![VMware vSphere Hypervisor program](assets/images/VMwareESXi_02.jpg){: width="1918" height="1010" loading="lazy"}

    <font size="+2">Prerequisites</font>

    As a starting point you need a **PC with a running VMware ESXi Hypervisor software** on which the DietPi system will run.

    <font size="+2">1. Download and extract the DietPi appliance image</font>

    Download the **DietPi ESXi** appliance image from [dietpi.com](https://dietpi.com/#download){:class="nospellcheck"} and unzip the downloaded file to a local folder. It is a _7z_ archive format so you will need to install either [7zip for Windows](https://www.7-zip.org/) or other alternative tools.

>     The zip file contains a couple of files, the important is the `DietPi_ESXi-x86_64-Bullseye.ova` file which have to be uploaded to the ESXi server.

Reviewer: ```suggestion
    The archive contains a couple of files, the important one is `DietPi_ESXi-x86_64-Bullseye.ova`, which has to be uploaded to the ESXi server.
```

    1. Tab **Select OVF and VMDK files**: Enter the name of your VM and drop or select the extracted `.ova` file. Then click **Next**.

        ![DietPi VM creation screenshot](assets/images/VMwareESXi_04.jpg){: width="942" height="556" loading="lazy"}

    1. Tab **Select storage**: Choose the location where the VM shall reside. Then click **Next**.
    1. Tab **Deployment options**: Set options according to your preferences and click **Next**.
    1. Tab **Ready to complete**: Check dialog and click **Finish**.

    <font size="+2">3. First boot of the new VMware image</font>

    Depending on your settings the VM automatically starts up and is shown in the VM management dialog. Otherwise click on the VM name.

    ![vSphere VM management screenshot 2](assets/images/VMwareESXi_05.jpg){: width="1669" height="859" loading="lazy"}

    You can open a console of your VM just by clicking on the small VM screen output to open a console window. Klick into the new console window to redirect your keyboard to this console.

    !!! info "Install VM tools"

        To install VM tools in your ESXi machine, it is an option to do it with the `open-vm-tools` via:

        ```sh

>         apt install open-vm-Tools

Reviewer: ```suggestion
        apt install open-vm-tools
```
I assume it's not a capital T, matching the text above?

        ```

>     !!! info "Boot times of a VM can be fast (less than 10 seconds)"

Reviewer: Do we need to add this as an info box? 😄

Author: It is so amazing. We should ask Holger to generate a video showing this.
Or we could add this in a blog in combination with our measured boot durations in https://dietpi.com/blog/?p=888 resp. https://dietpi.com/stats.html.

Author: Possibly we should change it into a warning? :-)

Author: ok, removed.
